esp-hal-common/src/rom/crc.rs

```diff
@@ -31,7 +32,7 @@
 //! crc = !crc32_be(crc, &data2); // finish
 //! ```
 //!
-//! # Examples
+//! ## Examples
```

Comment:
This isn't for you but for the esp-rs team, but I think these examples could use a bit of love. They could probably be merged into one, the definitions could be turned into comments and what's up with the `!0` and `!crc32_be` stuff? 🤔